msfrpcd

```diff
@@ -68,47 +177,67 @@ arguments.parse(ARGV) { |opt, idx, val|
       opts['URI'] = val
     when "-n"
       frameworkOpts['DisableDatabase'] = true
+    when "-j"
+      json_rpc = true
+    when "-k"
+      ws_ssl_key = val
+    when "-c"
+      ws_ssl_cert = val
+    when "-v"
+      ssl_enable_verify = true
     when "-h"
       print("\nUsage: #{File.basename(__FILE__)} <options>\n" +	arguments.usage)
       exit
+    end
+  }
+
+  $0 = "msfrpcd"
+
+  require 'msf/base'
+  require 'msf/ui'
+  require 'msf/util/service_helper'
+
+  begin
+    if json_rpc
+
+      if !File.file?(@ws_ssl_key_default) || !File.file?(@ws_ssl_cert_default)
+        $stdout.puts "[-] It doesn't appear msfdb has been run; please run 'msfdb init' first."
+        abort
+      end
+
+      $stderr.puts "[*] JSON-RPC starting on #{opts['ServerHost']}:#{opts['ServerPort']} (#{opts['SSL'] ? "SSL" : "NO SSL"})..."
+      $stderr.puts "[*] URI: /api/<version>/json-rpc"
```

Comment:
Could we specify the exact version here somehow? Or at least indicate the format is `v<num>`?

Reply:
It's dynamic, but given we really only have `v1` (`v10`) I'll just change it until a future `v2` is in use.